flashcards/backend/docs/decks.tests.rst

```diff
@@ -0,0 +1,21 @@
+decks.tests package
+===================
+
+Submodules
```

Comment:
remove the `Submodules` title it only makes a mess in documentation Please check for the subtitle in other .rst files